spec/pb_kits/playbook/props/numeric_spec.rb

```diff
@@ -0,0 +1,18 @@
+# frozen_string_literal: true
+
+RSpec.describe Playbook::Props::Numeric do
+  describe "#validate" do
+    it "returns true given a valid input", :aggregate_failures do
+      expect(Playbook::Props::Numeric.new.validate(nil)).to eq true
+      expect(Playbook::Props::Numeric.new.validate(12)).to eq true
+      expect(Playbook::Props::Numeric.new.validate(12.3)).to eq true
+    end
+
+    it "returns false given anything something besides a integer or float", :aggregate_failures do
```

Comment:
I think this was meant to be "anything" or "something"

Reply:
good catch